hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -537,26 +553,26 @@ ContainerCommandResponseProto> executePutBlock(boolean close,
     } catch (InterruptedException ex) {
       Thread.currentThread().interrupt();
       handleInterruptedException(ex, false);
+      // never reach, just to make compiler happy.
+      return null;
```

Comment:
is compiler complaining, or is it findbugs?

Reply:
It's compiler. It doesn't know that `handleInterruptedException()` always throws. Putting a `return` placeholder there makes it simpler than any other way I know.

Reply:
Good idea @szetszwo . Thanks for chiming in